localisation/wc_religions.csv

```diff
@@ -25,7 +25,7 @@ FEL;Fel;Fel;Fel;;;;;;;;;;;x;
 dragon_religion_group;Draconic;draconique;drakonisch;;;;;;;;;;;x;
 sect_of_the_dragons;Draconic;draconique;drakonisch;;;;;;;;;;;x;
 sect_of_the_dragons_DESC;The worship of dragons, reptilian creatures, usually winged, with magical or unusual abilities. A lot of creatures dedicated their lives to servitude under the great dragons. When dragons empower mortal servants they became dragonspawn and drakonid thanks to the dragons they served.;;;;;;;;;;;;;x;
-orcish_fel_DESC;Whether its power is found through contracts between warlocks and demonic lords or through the blood of a certain Annihilan, fel threatens to corrupt completely and absolutely. Though famous for its fel-fueled crusade, the Burning Legion is not responsible for all of those who find their way to fel corruption, and those who fall on their own find themselves craving power unending, usually until their very undoing. The fel-corrupted are forever hungry for power and control, even if it should cost them their soul or the lives of their people.;;Egal, ob seine Macht durch Vertr�ge zwischen Hexenmeistern und d�monischen F�rsten oder durch das Blut eines bestimmten Annihilaners erlangt wird, droht das Fel vollst�ndig und absolut zu verderben. Obwohl sie f�r ihren mit Fel-Magie befeuerten Kreuzzug bekannt ist, ist die brennende Legion nicht f�r alle verantwortlich, die den Weg zur Fel-Korruption finden, und diejenigen, die auf sich allein gestellt fallen, finden sich in einem unendlichen Verlangen nach Macht wieder, f�r gew�hnlich bis zu ihrem eigenen Verderben. Die Verderbten sind f�r immer hungrig nach Macht und Kontrolle, auch wenn es sie ihre Seele oder das Leben ihres Volkes kosten sollte.;;;;;;;;;;;;x
+orcish_fel_DESC;Whether its power is found through contracts between warlocks and demonic lords or through the blood of a certain Annihilan, fel threatens to corrupt completely and absolutely. Though famous for its fel-fueled crusade, the Burning Legion is not responsible for all of those who find their way to fel corruption, and those who fall on their own find themselves craving power unending, usually until their very undoing. The fel-corrupted are forever hungry for power and control, even if it should cost them their soul or the lives of their people.;;Egal, ob seine Macht durch Vertr�ge zwischen Hexenmeistern und d�monischen F�rsten oder durch das Blut eines bestimmten Annihilaners erlangt wird, droht das Fel vollst�ndig und absolut zu verderben. Obwohl sie f�r ihren mit Fel-Magie befeuerten Kreuzzug bekannt ist, ist die brennende Legion nicht f�r alle verantwortlich, die den Weg zur Fel-Korruption finden, und diejenigen, die auf sich allein gestellt fallen, finden sich in einem unendlichen Verlangen nach Macht wieder, f�r gew�hnlich bis zu ihrem eigenen Verderben. Die Verderbten sind f�r immer hungrig nach Macht und Kontrolle, auch wenn es sie ihre Seele oder das Leben ihres Volkes kosten sollte.;;;;;;;;;;;x;
```

Comment:
", and those who fall on their own find"
remove the 'and' or the ',' 
I prefer ", those who fall on their own find"

Reply:
not relevant to the branch